ipet/parsing/ReaderManager.py

```diff
@@ -226,12 +227,21 @@ def updateProblemName(self, line, currentcontext, readers):
         """
         sets up data structures for a new problem if necessary
         """
+        self.updateLineNumberData(line[0], currentcontext, "LineNumbers_Begin")
```

Comment:
This looks wrong. Previously, this code was at the end of updateProblemName() **on purpose**. With your change, you screw up the LineNumbersBegin for the previous problem.

Reply:
Really? I checked it when i moved it up and it seems fine, `updateLineNumberData` is only a wrapped `testrun.addData`, which is also used later. Also the Tests succeed.

Reply:
It is still a mystery to me, why, but it works. The problem that I thought would occur is that the datum is stored before the problem name is switched.